src/Ledger/GovernanceActions.lagda

```diff
@@ -99,34 +100,30 @@ Figure~\ref{defs:governance} defines several data types used to represent govern
 \begin{figure*}[h]
 \begin{code}[hide]
 private
-  ⟨_,_,_⟩ : {A : Set} → A → A → A → GovRole → A
-  ⟨ q₁ , q₂ , q₃ ⟩ = λ { CC → q₁ ; DRep → q₂ ; SPO → q₃ }
+  ∣_∣_∣_∣ : {A : Set} → A → A → A → GovRole → A
+  ∣ q₁ ∣ q₂ ∣ q₃ ∣ = λ { CC → q₁ ; DRep → q₂ ; SPO → q₃ }
 \end{code}
 \begin{code}
 threshold : PParams → Maybe ℚ → GovAction → GovRole → Maybe ℚ
 threshold pp ccThreshold' = λ where
-    NoConfidence           → ⟨ noVote            , vote P1      , vote Q1 ⟩
+    NoConfidence           → ∣ noVote            ∣ vote P1      ∣ vote Q1      ∣
     (NewCommittee _ _ _)   → case ccThreshold' of λ where
-      (just _)             → ⟨ noVote            , vote P2a     , vote Q2a ⟩
-      nothing              → ⟨ noVote            , vote P2b     , vote Q2b ⟩
-    (NewConstitution _ _)  → ⟨ vote ccThreshold  , vote P3      , noVote ⟩
-    (TriggerHF _)          → ⟨ vote ccThreshold  , vote P4      , vote Q4 ⟩
-    (ChangePParams x)      → ⟨ vote ccThreshold  , vote (P5 x)  , noVote ⟩
-    (TreasuryWdrl _)       → ⟨ vote ccThreshold  , vote P6      , noVote ⟩
-    Info                   → ⟨ vote 2ℚ           , vote 2ℚ      , vote 2ℚ ⟩
+      (just _)             → ∣ noVote            ∣ vote P2a     ∣ vote Q2a     ∣
+      nothing              → ∣ noVote            ∣ vote P2b     ∣ vote Q2b     ∣
+    (NewConstitution _ _)  → ∣ vote ccThreshold  ∣ vote P3      ∣ noVote       ∣
+    (TriggerHF _)          → ∣ vote ccThreshold  ∣ vote P4      ∣ vote Q4      ∣
+    (ChangePParams x)      → ∣ vote ccThreshold  ∣ vote (P5 x)  ∣ noVote       ∣
+    (TreasuryWdrl _)       → ∣ vote ccThreshold  ∣ vote P6      ∣ noVote       ∣
+    Info                   → ∣ vote defer        ∣ vote defer   ∣ vote defer   ∣
```

Comment:
@WhatisRT, is `vote defer` okay, or would you still prefer to see `vote*` in the table (as an alias for `vote defer`)?

Reply:
We can leave it like that for now and potentially change it later